collectors/proc.plugin/multi_metadata.yaml

```
@@ -2023,20 +2052,23 @@ modules:
       plugin_name: proc.plugin
       module_name: /proc/mdstat
       monitored_instance:
-        name: proc /proc/mdstat
+        name: Software RAID devices
```

```suggestion
        name: MDRAID
```

There are a bunch of soft-RAID implementations for Linux, we need to name the integrations based on the actual implementation.